tests/app/test_daemon.py

```diff
@@ -1,21 +1,24 @@
 #!/usr/bin/env python
 
 
-import pytest
-if pytest.PLATFORM == "win32":
-    pytest.skip("Unsupported Platform")
-
-
 import sys
-from os import kill
-from time import sleep
 from errno import ESRCH
+from os import kill
 from signal import SIGTERM
 from subprocess import Popen
+from time import sleep
 
+import pytest
 
 from . import app
 
+if pytest.PLATFORM == "win32":
+    pytest.skip("Unsupported Platform")
+
+
```

Comment:
Fixed, that said there is plenty of "too much whitespace" still around, but I am not going to fix all of those manually for this PR